Assets/Scripts/Battle/BattleSystem/BattleParticipant.cs

```diff
@@ -26,7 +26,9 @@ public abstract class BattleParticipant : MonoBehaviour
     /// <summary> Backing statuses list. Don't use this for most iteration. </summary>
     private List<Status> _statuses = new List<Status>();
     /// <summary> Use this statuses list for general iteration. It clones the list to avoid issues with statuses being removed during iteration. </summary>
-    private List<Status> statuses { get { return new List<Status>(_statuses); } }
+    public List<Status> statuses { get { return new List<Status>(_statuses); } }
+
+    public StatusDisplay StatusPanel;
```

Comment:
This is a UI element, so it should go under `BattleParticipantDisplay` rather than the backend `BattleParticipant`.